pkg/forwarder/forwarder.go

```diff
@@ -166,16 +166,24 @@ type DefaultForwarder struct {
 
 // NewDefaultForwarder returns a new DefaultForwarder.
 func NewDefaultForwarder(keysPerDomains map[string][]string) *DefaultForwarder {
+	return NewDefaultForwarderWithOptions(keysPerDomains, true)
+}
+
+// NewDefaultForwarderWithOptions creates a new DefaultForwarder with the given configuration
```

Comment:
I guess you are not going to use the esoteric way to set checker by returning a function that modifies the new service object? :)